src/components/login/login.component.ts

```diff
@@ -20,7 +20,7 @@ export class LoginComponent {
               private router: Router) {
     // Tests if user already logged in
     this.backend.sendLoginData("", "").subscribe(res => {
-      if (res.error == "already logged in") this.router.navigate(['/create-game']);
+      if (res.status == 409) this.router.navigate(['/create-game']);
```

Comment:
Yes, this change was breaking, but if you had reviewed the corresponding PR in the server, this would be done now